plugins/api/src/get_file_descriptor_info.cpp

```diff
@@ -189,7 +189,7 @@ namespace
         return {
             {"host_name", to_json(_p->hostName)},
             {"rcat_enabled", _p->rcatEnabled},
-            {"re_host_flag", _p->reHostFlag},
+            {"re_host_flag", _p->reHostFlag}, // TODO Is this referring to the delay server?
```

Comment:
Thoughts?

Reply:
yes, it is referring to the delay server.

but we are no longer using this mechanism (a flag at the server level), so if you deem it safe to remove from this structure, you can.  if not, then we should just mark it 'dummy' and/or 'deprecated'.

Reply:
removed it in favor of https://github.com/irods/irods/pull/6274#discussion_r837866851